pom.xml

```diff
@@ -22,14 +22,14 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>dynamic-spark</artifactId>
-  <version>2.2.2</version>
+  <version>2.3.0-SNAPSHOT</version>
 
   <properties>
     <!-- properties for script build step that creates the config files for the artifacts -->
     <data.pipeline.parent>system:cdap-data-pipeline[6.0.0-SNAPSHOT,7.0.0-SNAPSHOT)</data.pipeline.parent>
```

Comment:
Change this to 6.1.1

Reply:
change what to 6.1.1? the pipeline and stream . parents ?

Reply:
should we go back and make the same changes in develop as well since this is a cherrypick?

Reply:
yea we can keep them same in develop as well as release. 
`[61.1,7.0.0-SNAPSHOT)`